### PR TITLE
fix(export): [#RBS-120] fix export PDF for multiple bookings per slot

### PR DESCRIPTION
**Pull request by qperie**
## Describe your changes
Adjust export PDF for daily and weekly wiew by displaying booking width according to their siblings (how many other bookings are overlapping).

## Issue ticket number and link
RBS-120 : https://jira.support-ent.fr/browse/RBS-120

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -23,4 +23,4 @@ junitVersion=4.12
 powerMockVersion=2.0.2
 mockitoVersion=2.+
 
-entCoreVersion=4.6-SNAPSHOT
+entCoreVersion=4.5.6
```
Comment (sjollois) [inline gradle.properties]: à faire dans un autre commit de type chore() non ?

**Review by alicedraillard — APPROVED** [review]

**Review by sjollois — APPROVED** [review]

**Review by Rseuret — APPROVED** [review]

File: src/main/java/net/atos/entng/rbs/service/pdf/JsonDayFormatter.java [hunk 2]
```diff
@@ -30,64 +34,67 @@ public JsonObject format() {
 		convertedObject.put(EDITION_DATE_FIELD_NAME, DateTime.now().toString("dd/MM/YYYY"));
```
Comment (BastienBenard) [inline src/main/java/net/atos/entng/rbs/service/pdf/JsonDayFormatter.java]: Ta fonction est plutôt long, est ce que on ne peut pas la split en plusieurs fonctions pour améliorer la lisibilité ?

File: src/main/java/net/atos/entng/rbs/service/pdf/JsonFormatter.java [hunk 3]
```diff
@@ -130,4 +133,65 @@ public static JsonFormatter buildFormater(JsonObject jsonExportResponse, String
...
+	 * Build the slot raw title list
+	 *
+	 * @param firstSlotOfADay: First time slot begin time
+	 * @param lastSlotOfADay:  Last time slot end time
+	 * @return The slot raw title list
+	 */
+	protected static JsonArray buildSlotRawTitles(DateTime firstSlotOfADay, DateTime lastSlotOfADay) {
+		JsonArray slotRawTitles = new JsonArray();
+		for (int i = firstSlotOfADay.getHourOfDay(); i < lastSlotOfADay.getHourOfDay(); i++) {
+			slotRawTitles.add(new JsonObject().put("value", i + ":00 - " + (i + 1) + ":00"));
```
Comment (BastienBenard) [inline src/main/java/net/atos/entng/rbs/service/pdf/JsonFormatter.java]: magic string value

**Review by BastienBenard — APPROVED** [review]

File: src/main/java/net/atos/entng/rbs/service/pdf/JsonDayFormatter.java (hunk 2 above)
Comment (flmartineau) [inline src/main/java/net/atos/entng/rbs/service/pdf/JsonDayFormatter.java]: Le `dd/MM/YYYY` vous l'avez pas dans une variable ?

**Review by flmartineau — APPROVED** [review]

**Pull request merged**